SI/Sem.02/Solutions/Task07.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	char letter;
+	cin >> letter;
+	if (letter >= 65 && letter <= 90)
```

Comment:
let us work with 'a' and 'z'.
In terms of functionality & performance -> the same.
In terms of readability, however, they are better.